definitions/private-law/json/CaseEvent.json

```diff
@@ -1721,5 +1721,18 @@
     "SecurityClassification": "Public",
     "ShowEventNotes": "N",
     "EventEnablingCondition": "caseTypeOfApplication=\"FL401\""
+  },
+  {
+    "LiveFrom": "01/07/2021",
+    "CaseTypeID": "PRLAPPS",
+    "ID": "citizen-case-submit-with-hwf",
```

Comment:
can we align this id to the existing ones like citizenCaseSubmitWithHWF, if possible please.

Reply:
Done